### DISCUSSION TITLE
Primefaces JDK Compatibility Java 8 vs Java 11

### DISCUSSION THREAD

**Discussion by xxaaxxaa** [primefaces/community#2567] [PrimeFaces] [answered]
Hi,

What is the JDK compatibility for Primefaces ?

1. Are the LTS Primefaces v12 v13 v14 compatible with Java 8 ? 
2. Possible to retain compatibilty with Java 8 or what the workaround to continue with Java 8 for any of these versions ?
3. What is the plan going forward ?

**Comment by jepsar** [comment, marked as answer]
PrimeFaces 14 is not a LTS version.

PrimeFaces 14 is not compatible with Java 8. Older versions are.

See https://github.com/primefaces/primefaces?tab=readme-ov-file#versions

You might expect compatibility bumps in newer versions in line with what you have seen on older versions.

**Reply by melloware** [reply]
Follow up that PF 13.0.X will be the last Java8 version permantly.  PF14+ will use JDK11 and eventually JDK17.
